tools/check_doc_approval.py

```diff
@@ -0,0 +1,76 @@
+import os
+import ast
+import hashlib
+import importlib
+import paddle.fluid
+
+
+files = [
+    "paddle.fluid.average",
+    "paddle.fluid.backward",
+    "paddle.fluid.clip",
+    "paddle.fluid.data_feeder",
+    "paddle.fluid.executor",
+    "paddle.fluid.initializer",
+    "paddle.fluid.io",
+    "paddle.fluid.layers",
+    "paddle.fluid.metrics",
+    "paddle.fluid.nets",
+    "paddle.fluid.optimizer",
+    "paddle.fluid.profiler",
+    "paddle.fluid.recordio_writer",
+    "paddle.fluid.regularizer",
+    "paddle.fluid.transpiler"
```

Comment:
lack of `paddle.fluid`?

Reply:
Oh !  I think you are right, I miss the  paddle.fluid .